inspirehep/modules/literaturesuggest/static/js/literaturesuggest/literature_submission_form.js

```diff
@@ -612,6 +614,13 @@ define(function(require, exports, module) {
       });
     },
 
+    // Collapse the links panel by default if the record is imported from arXiv
+    collapseLinksPanel: function collapseLinksPanel() {
+      if($.trim(this.$arxiv_id_field.val().length) > 0) {
+        $('#collapse-3').prev().find('a.panel-toggle').click();
```

Comment:
I checked manually that this works, but code like this is **really** brittle... Unfortunately, we will have to live with this until we will have time to refactor the forms.